<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WarningErrorMsgClass</class>
 <widget class="QWidget" name="WarningErrorMsgClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>152</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WarningErrorMsg</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: #2a2a2a;
	border: 3px solid #3b3e41;
	border-radius: 3px;
}

QLabel {
	border: 0px;
}

QPushButton#btnAccept {
    background-color: #3b3e41;
    border-radius: 3px;
    border: 1px solid #757575;
    font: bold 12px;
	color: white;
    min-width: 6em;
    padding: 6px;
}

QPushButton#btnAccept:hover {
    background-color: #85b448;
	cursor: pointer;
}

QPushButton#btnAccept:pressed {
    background-color: #d86a39;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="messageIcon">
         <property name="minimumSize">
          <size>
           <width>96</width>
           <height>96</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>96</width>
           <height>96</height>
          </size>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;img src=&quot;Resources/warningIcon.png&quot;/&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="messageTitle">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: white; margin-left: 5px; margin-top: 5px;</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="messageContent">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: white; margin-left: 5px; </string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="btnAccept">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Close window</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="WarningErrorMsg.qrc"/>
 </resources>
 <connections/>
</ui>
